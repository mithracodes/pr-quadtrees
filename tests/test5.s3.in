144.97056424489568 -37.796155887263744
144.96941668057087 -37.79606116572821
144.97550677200553 -37.79327234020523
144.97531546208762 -37.794366870830814
