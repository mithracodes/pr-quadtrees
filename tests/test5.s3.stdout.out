144.97056424489568 -37.796155887263744 --> SW SE
144.96941668057087 -37.79606116572821 --> SW SW
144.97550677200553 -37.79327234020523 --> NE
144.97531546208762 -37.794366870830814 --> SE
